_includes/embed/video.html

```diff
@@ -31,8 +33,35 @@
 {% endif %}
 
 <p>
-  <video class="embed-video file" src="{{ video_url }}" {{ poster }} {{ attributes }}>
-    Your browser doesn't support HTML video. Here is a <a href="{{ video_url }}">link to the video</a> instead.
+  <video class="embed-video file" {{ poster }} {{ attributes }}>
+    {% assign parts = video_url | split: '/' %}
+    {% assign filename = parts | last %}
+    {% assign extension = filename | split: '.' | last %}
+    {% assign basename = filename | split: '.' | pop | join: '.' %}
```

Comment:
```suggestion
    {% assign basename = filename | split: '.' | first %}
```

Reply:
This option will only work if there are no more dots in the filename, for example: `audio.mp3`. But my implementation supports more complex paths, for example: `audio.ai.mp3`. In this case I get exactly the name with a suffix that the user can use.

Reply:
That makes sense